src/Kernel/GlobalVariable.class.st

```diff
@@ -33,6 +33,5 @@ GlobalVariable >> isGlobalVariable [
 
 { #category : #accessing }
 GlobalVariable >> scope [
-	"The scope of Globals is Smalltalk globals"
-	^ Smalltalk globals
+	^ self definingClass environment
```

Comment:
I wonder that there is definingClass for globals. What is that?

Reply:
Ups, true, that makes not sense. I will put it back for now to a hard-coded global environment. (I am reluctant to add an environment pointer to GlobalVariable as it would cost some memory...)